tests/cc/test_bpf_table.cc

```diff
@@ -67,6 +67,37 @@ TEST_CASE("test bpf table", "[bpf_table]") {
   REQUIRE(res.code() != 0);
 }
 
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(4, 6, 0)
+TEST_CASE("test bpf percpu tables", "[bpf_percpu_table]") {
+  const std::string BPF_PROGRAM = R"(
+    BPF_TABLE("percpu_hash", int, u64, myhash, 128);
+  )";
+
+  ebpf::BPF *bpf(new ebpf::BPF);
```

Comment:
Do you want to use `ebpf::BPF bpf` instead of the above `new` operator? This way, you won't need to call `delete`.